job-service-acceptance-tests/src/test/java/com/hpe/caf/jobservice/acceptance/JobServiceDatabaseUtil.java

```diff
@@ -95,8 +96,8 @@ public static void assertJobRowExists(final String jobId) throws SQLException
             st.setString(1, jobId);
             final ResultSet jobRS = st.executeQuery();
             jobRS.next();
-            Assert.assertEquals(jobRS.getString(1), jobId);
-            Assert.assertTrue(!jobRS.next());
+            Assert.assertEquals(jobRS.getString(1), jobId, "Job "+jobId+" does not exist in job table");
+            Assert.assertTrue(!jobRS.next(), "Duplicate job "+jobId+" should not have been existed in job table");
```

Comment:
Reword this message, does not make sense.
```suggestion
            Assert.assertTrue(!jobRS.next(), "Duplicate job "+jobId+" should not exist in job table");
```